install_config/configuring_extended_ldap_attributes.adoc

```diff
@@ -0,0 +1,175 @@
+= Configuring Extended LDAP Attributes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+This topic builds upon xref:../install_config/ssd_for_ldap_failover.adoc[Setting
+up SSSD for LDAP Failover] and
+xref:../install_config/configuring_form_based_authentication.adoc[Configuring
+Form-Based Authentication] and focused on configuring extended Lightweight
+Directory Access Protocol (LDAP) attributes.
+
```

Comment:
This should probably not be called an extended attribute in the context of OpenShift documentation. What it means in the SSSD context is "any attribute stored in LDAP that isn't required to satisfy a lookup for POSIX user information".

For OpenShift, the only additional attribute we care about is the email address. So probably this should be rephrased to something more like: "SSSD normally only retrieves information about the username and display name, so we need to configure it to also retrieve the user's email address."